data/js/app.js

```diff
@@ -93,6 +93,22 @@ const vueApp = Vue.createApp({
 
                 fetch("/toggleScaleCalibration", requestOptions)
             }
+        },
+
+        async confirmReset() {
+            const confirmed = window.confirm(
+                "Are you sure you want to reset the WiFi settings?\n\nThis will erase saved credentials and restart the device."
+            );
+
+            if (!confirmed) return;
+
+            try {
+                const response = await fetch("/reset", { method: "POST" });
```

Comment:
Suggestion: This would probably be better called /wifiReset or similar, it's not resetting all configuration or only restarting the device.